src/css/flags/breakpoints.less

```diff
@@ -17,11 +18,19 @@ Breakpoints (px):
   }
 }
 
+.jw-flag-touch.jw-breakpoint-3 {
+  .jw-text-elapsed,
+  .jw-text-duration {
+    display: none;
+  }
+}
+
 .jw-flag-touch.jw-breakpoint-2,
 .jw-flag-touch.jw-breakpoint-1,
 .jw-flag-touch.jw-breakpoint-0 {
-    .jw-text-elapsed,
-    .jw-text-duration {
-      display: none;
-    }
+  .jw-text-elapsed,
+  .jw-text-duration,
+  .jw-nextup-container {
+    display: none;
+  }
```

Comment:
Do we need the .jw-flag-touch flag as part of these rules? What does it look like on desktop browsers?